test/system/030-run.bats

```diff
@@ -58,6 +58,25 @@ echo $rand        |   0 | $rand
     is "$output" ".*invalidflag" "failed when passing undefined flags to the runtime"
 }
 
+# bats test_tags=ci:parallel
+@test "podman run - containers.conf runtime options" {
+    skip_if_remote "requires local containers.conf"
+
+    containersConf=$PODMAN_TMPDIR/containers.conf
+    cat >$containersConf <<EOF
+[engine]
+runtime="crun"
```

Comment:
@ricardobranco777 Do you have crun installed on your test system? I think this must use `$(podman_runtime)` instead to use the proper test runtime so the right error can be triggered